syntax = "proto2";

package followifier;

message ESP32Message {
    required string apMac = 1; // FIXME change name
    required string ssid = 2;
    required int64 timestamp = 3; // TODO check whether this is set by the phone
    required string frame_hash = 4; // TODO is it computed starting from AP's MAC & timestamp?
    required int32 rsi = 5;
}

message Batch {
    required string boardMac = 1; // FIXME change name
    repeated ESP32Message messages = 2;
}
